devicehive-admin/Dockerfile

```diff
@@ -11,7 +11,8 @@ ADD admin-start.sh /opt/devicehive/
 ADD https://github.com/devicehive/devicehive-admin-console/archive/stable.tar.gz /tmp
 RUN tar -C /tmp -xzf /tmp/stable.tar.gz \
     && cp -r /tmp/devicehive-admin-console-stable/* /opt/devicehive/admin/ \
```

Comment:
`/tmp/devicehive-admin-console-stable/` also should be removed. This directory is just copied and as a result we have 2 identical directories in image